src/SourceBuild/content/repo-projects/deployment-tools.proj

```diff
@@ -9,9 +9,11 @@
     <RepositoryReference Include="source-build-reference-packages" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(UseOfficialBuildVersioning)' != 'false'">
```

Comment:
Hey @jkoritzinsky, do you recall the reason for adding this condition? Is there something that breaks if deployment-tools knows about the git hash in official building versioning mode? Any idea how I can observe this locally?

Reply:
I think I did this because (at least to the best of my memory), I didn't want the VMR's git commit hash to be treated as the commit for deployment tools.

Alternatively, I did this because the git commit hash info is only provided by the git-info folder in the VMR, which we don't import in dev-versioning mode (and eventually will cease to exist once we ship out of the VMR).

Things might work with this condition removed, I'm not sure.

Reply:
Thanks!